apps/trading/components/footer/footer.tsx

```diff
@@ -27,19 +28,36 @@ export const NodeHealth = () => {
   const onClick = useCallback(() => {
     setNodeSwitcher(true);
   }, [setNodeSwitcher]);
+  const incidentsLink = (
+    <ExternalLink className="ml-1" href={MAINNET_INCIDENT_LINK}>
+      {t('Mainnet status & incidents')}
+    </ExternalLink>
+  );
   return VEGA_URL ? (
-    <FooterButton onClick={onClick} data-testid="node-health">
-      <FooterButtonPart>
-        <HealthIndicator text={text} intent={intent} />
-      </FooterButtonPart>
-      <FooterButtonPart>
-        <NodeUrl url={VEGA_URL} />
-      </FooterButtonPart>
-      <FooterButtonPart>
-        <span title={t('Block height')}>{datanodeBlockHeight}</span>
-      </FooterButtonPart>
-    </FooterButton>
-  ) : null;
+    <>
```

Comment:
suggestion (non blocking): rather than rendering the link twice, in both true _and_ false cases of the ternary check, I think lift the fragment so it can be rendered once in a single place

```
return (
  <>
    {VEGA_URL && (
      <FooterButton>...</FooterButton>
    )}
    <ExternalLink className="ml-1" href={MAINNET_INCIDENT_LINK}>
      {t('Mainnet status & incidents')}
    </ExternalLink>
  </>
)
```